nifi-toolkit/nifi-toolkit-encrypt-config/src/main/groovy/org/apache/nifi/properties/ConfigEncryptionTool.groovy

```diff
@@ -385,26 +370,26 @@ class ConfigEncryptionTool {
                 }
             }
 
-            if (commandLine.hasOption(FLOW_XML_ARG)) {
-                if (isVerbose) {
-                    logger.info("Handling encryption of flow.xml.gz")
-                }
-                flowXmlPath = commandLine.getOptionValue(FLOW_XML_ARG)
-                outputFlowXmlPath = commandLine.getOptionValue(OUTPUT_FLOW_XML_ARG, flowXmlPath)
-                handlingFlowXml = true
-
-                newFlowAlgorithm = commandLine.getOptionValue(NEW_FLOW_ALGORITHM_ARG)
-                newFlowProvider = commandLine.getOptionValue(NEW_FLOW_PROVIDER_ARG)
-
-                if (flowXmlPath == outputFlowXmlPath) {
-                    // TODO: Add confirmation pause and provide -y flag to offer no-interaction mode?
-                    logger.warn("The source flow.xml.gz and destination flow.xml.gz are identical [${outputFlowXmlPath}] so the original will be overwritten")
-                }
-
-                if (!commandLine.hasOption(NIFI_PROPERTIES_ARG)) {
-                    printUsageAndThrow("In order to migrate a flow.xml.gz, a nifi.properties file must also be specified via '-n'/'--${NIFI_PROPERTIES_ARG}'.", ExitCode.INVALID_ARGS)
-                }
-            }
+//            if (commandLine.hasOption(FLOW_XML_ARG)) {
+//                if (isVerbose) {
+//                    logger.info("Handling encryption of flow.xml.gz")
+//                }
+//                flowXmlPath = commandLine.getOptionValue(FLOW_XML_ARG)
+//                outputFlowXmlPath = commandLine.getOptionValue(OUTPUT_FLOW_XML_ARG, flowXmlPath)
+//                handlingFlowXml = true
+//
+//                newFlowAlgorithm = commandLine.getOptionValue(NEW_FLOW_ALGORITHM_ARG)
+//                newFlowProvider = commandLine.getOptionValue(NEW_FLOW_PROVIDER_ARG)
+//
+//                if (flowXmlPath == outputFlowXmlPath) {
+//                    // TODO: Add confirmation pause and provide -y flag to offer no-interaction mode?
+//                    logger.warn("The source flow.xml.gz and destination flow.xml.gz are identical [${outputFlowXmlPath}] so the original will be overwritten")
+//                }
+//
+//                if (!commandLine.hasOption(NIFI_PROPERTIES_ARG)) {
+//                    printUsageAndThrow("In order to migrate a flow.xml.gz, a nifi.properties file must also be specified via '-n'/'--${NIFI_PROPERTIES_ARG}'.", ExitCode.INVALID_ARGS)
+//                }
+//            }
```

Comment:
This block of code needs to be uncommented and working. Although the code says `flow.xml.gz`, it also works with `flow.json.gz`, so names should be adjusted accordingly.

Reply:
As noted in relation to the arguments, it looks like this can be uncommented for this PR, and then other changes can be handled separately.